.github/ISSUE_TEMPLATE.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+literal_block: |
+  > If you have encountered a security-related issue with Yii, please use the [security issue form](https://www.yiiframework.com/security) to report any security issues you have encountered to us.
+  > ***WARNING!*** DO NOT use the issue tracker or discuss it in the public forum as it will cause more damage than help.
+description: Create a bug report
+labels: ["defect"]
+body:
+  - type: input
+    id: yiiversion
+    attributes:
+      label: Yii Version
+      description: Which version of Yii are you using?
+      placeholder: "Enter the version you are using of this package"
+    validations:
+      required: true
+  - type: input
+    id: phpversion
+    attributes:
+      label: Version PHP
+      description: What version of PHP are you running?
+      placeholder: "Enter the PHP version you are using"
+    validations:
+      required: true
+  - type: dropdown
+    id: osversion
+    attributes:
+      label: Operating System
+      description: What version of operating system are you running?
+      placeholder: "Linux (Default)"
+      multiple: false
+      options:
+        - Microsoft Windows
+        - Apple macOS
+        - Linux (Default)
+        - Android
+        - iOS
+        - Chrome OS
```

Comment:
```suggestion
PHP doesn't really run on these. We should remove it.
```

Reply:
add "PHP doesn't really run on these. We should remove it." ?

Reply:
done. review